dev-python/os-api-ref/os-api-ref-2.2.0.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="8"
+
+PYTHON_COMPAT=( python3_{8..9} )
+
+inherit distutils-r1
+
+DESCRIPTION="Sphinx Extensions to support API reference sites in OpenStack"
+HOMEPAGE="https://docs.openstack.org/os-api-ref/latest"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	>=dev-python/openstackdocstheme-2.2.1[${PYTHON_USEDEP}]
+	>=dev-python/pyyaml-3.1.2[${PYTHON_USEDEP}]
+	>=dev-python/six-1.10.0[${PYTHON_USEDEP}]
+	dev-python/sphinx[${PYTHON_USEDEP}]
+"
+DEPEND="
+	${RDEPEND}
+	>=dev-python/pbr-2.0.0[${PYTHON_USEDEP}]
+	!~dev-python/pbr-2.1.0[${PYTHON_USEDEP}]
```

Comment:
We don't have ancient pbr versions, so you can just dep on >=2.2.0.

Reply:
Is there a reason we don't match upstream's requirements files?  I'd like to see something from QA that says not to do this as it's a safety issue (who knows what overlays are there).

Reply:
removed